### ISSUE TITLE
Accessing certain skia properties crashes app

### ISSUE THREAD

**Issue by pusolito** [JetBrains/skiko#431]
Any call to `Canvas.localToDeviceAsMatrix33` causes the app to core dump. This is very easy to reproduce. Simply add the following line to `onRender` within `ClocksJvm` in the SkiaJvmSample. The same is true for calls to `Typeface.fontFamilyName` and `Typeface.fontFamilyNames`.

```kotlin
// ...

override fun onRender(canvas: Canvas, width: Int, height: Int, currentTimestamp: Long) {
    canvas.localToDeviceAsMatrix33 <--- Crash
    Font(Typeface.makeFromName("Courier", FontStyle(300, 5, FontSlant.UPRIGHT)), 13f).typefaceOrDefault.familyNames <--- Crash
    // ...   
}

// ...
```

See attached core dump for more details.

[hs_err_pid10301.log](https://github.com/JetBrains/skiko/files/7731875/hs_err_pid10301.log)

**Comment by olonho** [comment]
Hmm, doesn't seem to crash for me. Does it crash with other JVMs?

**Comment by olonho** [comment]
OK, can reproduce on M1 ARM with `Zulu15.28+1013-CA`.

**Comment by olonho** [comment]
Typo, can be fixed like this 
```
diff --git a/skiko/src/commonMain/kotlin/org/jetbrains/skia/Canvas.kt b/skiko/src/commonMain/kotlin/org/jetbrains/skia/Canvas.kt
index 3e9d6713..5f3aaefb 100644
--- a/skiko/src/commonMain/kotlin/org/jetbrains/skia/Canvas.kt
+++ b/skiko/src/commonMain/kotlin/org/jetbrains/skia/Canvas.kt
@@ -1630,7 +1630,7 @@ private external fun _nDrawPaint(ptr: NativePointer, paintPtr: NativePointer)
 private external fun _nSetMatrix(ptr: NativePointer, matrix: InteropPointer)
 
 @ExternalSymbolName("org_jetbrains_skia_Canvas__1nGetLocalToDevice")
-private external fun _nGetLocalToDevice(ptr: NativePointer, resultFloats: InteropPointer): FloatArray
+private external fun _nGetLocalToDevice(ptr: NativePointer, resultFloats: InteropPointer)
 
 @ExternalSymbolName("org_jetbrains_skia_Canvas__1nResetMatrix")
 private external fun _nResetMatrix(ptr: NativePointer)
```

**Comment by pusolito** [comment]
Is this fix something that could make it into the next release?

**Comment by olonho** [comment]
Sure, we'll fix these problems.

**Comment by olonho** [comment]
Shall be fixed with Skiko 0.7.0.
